translator/translate/otel/exporter/awscloudwatch/translator.go

```diff
@@ -164,6 +164,14 @@ func getDropOriginalMetrics(conf *confmap.Conf) map[string]bool {
 		*/
 		if dropMetrics := common.GetArray[any](conf, dropOriginalCfgKey); dropMetrics != nil {
 			for _, dropMetric := range dropMetrics {
+				if category == "collectd" || category == "statsd" || category == "ethtool" {
```

Comment:
nitpick - maybe create a map of these and check if the category matches any key in the map, makes it more readable

Reply:
Fixed.